src/styles/blocks/header.scss

```diff
@@ -0,0 +1,50 @@
+.container {
+  margin: 0 auto;
+  padding: 24px 18px 0;
+  max-width: 1020px;
+
+  @include onTablet {
+    padding: 51px 72px 0;
+  }
+}
+
+.header {
+  box-sizing: border-box;
+  height: 100vh;
+
+  background-image: url(../images/met_header-image.svg);
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+
+  &__top-row-layout {
+    margin: 0 0 129px;
+
+    @include onTablet {
+      margin-bottom: 141px;
+    }
+
+    @include onDesktop {
+      margin-bottom: 84px;
+    }
+  }
+
+  &__title {
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 100%;
```

Comment:
Consider using a variable for the font-weight property to maintain consistency across the project.